### PR TITLE
add mongo directory and bboard_v1.json and update readme

### PR DESCRIPTION
**Pull request by PaulApivat**
**What?**

1. Added mongo directory with bboard_v1.json file with latest json structure of bounty card. 
2. Added links to Readme.md

**Why?**

1. To prepare for simulating Bounty card data.

**How?**

1. Went through Whimsical and Possible Queries documents to figure out needed key-value pairs.

**Testing?**

1. No testing has been done.

### CODE REVIEW

File: mongo/bboard_v1.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+[
+    {
+        "//": "Comment: bountyHash message -> bankless dao bounty board"
+    },
+    {
+        "season" : 2,
+        "bountyTitle": "A new About Us webpage",
+        "bountyDescription": "Description of webpage",
+        "doneCriteria": "A completed bounty has the following...",
```
Comment (Behold3th) [inline mongo/bboard_v1.json:9]: we could change to bountyCriteria for consistency of field names

File: mongo/bboard_v1.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            {"guildName": "Treasury Guild"},
+            {"guildName": "Developer's Guild"},
+            {"guildName": "Analytics Guild"},
+            {"guildName": "Writer's Guild"}
+        ],
+        "bountyCreatedBy": {
+            "isDaoMember" : true,
+            "guildName" : "Analytics Guild",
+            "discordHandle" : "@paulapivat#8888",
+            "publicAddress" : "0x2d94aa3e47d9d5024503ca8491fce9a2fb4da198"
```
Comment (Behold3th) [inline mongo/bboard_v1.json:25]: Including the wallet address will be great for MVP 2 and future iterations, for now I think we could leave this out as bounties will be distributed through the social layer for MVP 1.

File: mongo/bboard_v1.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        },
+        "bountyCreatedAt": "2021-07-10T13:49:51.141Z",
+        "bountyDueAt" : "2021-09-11T13:49:51.141",
```
Comment (Behold3th) [inline mongo/bboard_v1.json:28]:
```suggestion
        "bountyDueAt" : "2021-09-11T13:49:51.141",
        "bountyImage" : "https://pbs.twimg.com/profile_images/1389400052448247816/qsOU0pih_400x400.jpg"
```

File: mongo/bboard_v1.json (hunk 1 above)
Comment (PaulApivat) [reply mongo/bboard_v1.json:9]: Sounds good, changes made, will push.

File: mongo/bboard_v1.json (hunk 2 above)
Comment (PaulApivat) [reply mongo/bboard_v1.json:25]: Ok, will take out "publicAddress" field for now.

File: mongo/bboard_v1.json (hunk 3 above)
Comment (PaulApivat) [reply mongo/bboard_v1.json:28]: Adding "bountyImage" to document.

**Pull request merged**